src/constants/tags.json

```diff
@@ -1,6 +1,6 @@
 [
-  { "value": "Technology", "label": "Technology" },
-  { "value": "Poetry", "label": "Poetry" },
-  { "value": "Films", "label": "Films" },
-  { "value": "World Politics", "label": "World Politics" }
+  { "value": " Technology ", "label": "Technology" },
+  { "value": " Poetry ", "label": "Poetry" },
+  { "value": " Films ", "label": "Films" },
+  { "value": " World Politics ", "label": "World Politics" }
```

Comment:
why space?

Reply:
If I select multiple tags, then it shows one after another, like one word. I tried to handle it with scss to give a space, but there is no scss element found for this as it came from the json file dynamically.